requirements.txt

```diff
@@ -1,13 +1,9 @@
 Django==3.2.6
 gunicorn==20.1.0
 PyPDF2[crypto]
-Pillow
-requests
-pdf2image
 certifi
 chardet>=3.0.4
 eyed3
-ipython
```

Comment:
good drop of dependencies. but im shocked we can drop pillow

Reply:
I think these were all listed twice.

Reply:
ah